File: app/src/main/java/com/artmurka/artmurkaapp/Presenter/InterfacesPresenter/ICheckoutPresenter.java
package com.artmurka.artmurkaapp.Presenter.InterfacesPresenter;


public interface ICheckoutPresenter {
    void getData();
    void postCheckout(String telephone, String message, String email, String pay, String delivery);
    void cityChanged(String msg);
    Boolean isEmailValid(String email);
    boolean isValidPhone(String phone);
    void selectCity(Integer cityPostition);
    void getAreas();
    void getCities(String cityName);
    void selectWarehouse(Integer warehousePosition);
    void detachView();
}

File: app/src/main/java/com/artmurka/artmurkaapp/presenter/interfaces_presenter/ICheckoutPresenter.kt
package com.artmurka.artmurkaapp.presenter.interfaces_presenter


interface ICheckoutPresenter {
    fun getData()
    fun postCheckout(telephone: String, message: String, email: String, pay: String, delivery: String)
    fun cityChanged(msg: String)
    fun isEmailValid(email: String): Boolean?
    fun isValidPhone(phone: String): Boolean
    fun selectCity(cityPostition: Int?)
    fun getAreas()
    fun getCities(cityName: String)
    fun selectWarehouse(warehousePosition: Int?)
    fun detachView()
}
